### PR TITLE
increase verbosity of init-video, readme updates

### PR DESCRIPTION
**Pull request by aptalca**

### CODE REVIEW

**Comment by aptalca** [comment]
Looks like the upstream repo is missing the nightly arm64 package. amd64 builds fine

File: readme-vars.yml [hunk 1]
```diff
@@ -14,82 +14,97 @@ available_architectures:
...
-  - {vol_path: "/data/tvshows", vol_host_path: "path/to/tvseries", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
+  - {vol_path: "/data/tvshows", vol_host_path: "/path/to/tvseries", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
   - {vol_path: "/data/movies", vol_host_path: "/path/to/movies", desc: "Media goes here. Add as many as needed e.g. `/data/movies`, `/data/tv`, etc."}
 param_usage_include_ports: true
 param_ports:
   - {external_port: "8096", internal_port: "8096", port_desc: "Http webUI."}
 param_usage_include_env: true
 param_env_vars:
-  - {env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use EG Europe/London"}
+  - {env_var: "TZ", env_value: "Europe/London", desc: "Specify a timezone to use (e.g. Europe/London)."}
```
Comment (Roxedus) [inline readme-vars.yml:31]: The TZ var is templated, and this value is skipped.

**Review by Roxedus — APPROVED** [review]

File: readme-vars.yml (hunk 1 above)
Comment (aptalca) [reply readme-vars.yml:31]: Gotcha, I copied from the master branch readme. I'll remove on both

**Comment by aptalca** [comment]
https://github.com/jellyfin/jellyfin-packaging/issues/19

**Review by Roxedus — APPROVED** [review]

**Pull request merged**